services/install-pihole.sh

```diff
@@ -3,6 +3,8 @@
 function install {
   # create service directory
   mkdir -p /srv/pihole
+  service dnsmasq stop
```

Comment:
@kjong why should install stop `dnsmasq`

Reply:
From what I've found, both pi-hole and dnsmasq use port 53, causing conflicts.
Also, pi-hole includes dnsmasq.